time/src/main/java/io/spine/time/OffsetTimes.java

```diff
@@ -93,15 +94,18 @@ public static OffsetTime of(java.time.OffsetTime value) {
      */
     public static String toString(OffsetTime value) {
         checkNotNull(value);
-        return toJavaTime(value).toString();
+        return TimeStringifiers.forOffsetTime()
+                               .convert(value);
     }
 
     /**
      * Parse from ISO 8601 string to {@code OffsetTime}.
```

Comment:
`ISO-8601` — as I understand you tried to use this notation throughout the changeset. There are still several more `ISO 8601` occurrences; to keep the comments cleaner I am not commenting each entry.

It's interesting that Wikipedia lists it as [`ISO 8601`](https://en.wikipedia.org/wiki/ISO_8601), so [does](https://www.iso.org/standard/40874.html) the ISO website. While JDK 8 Javadocs [do a hyphen version](https://docs.oracle.com/javase/8/docs/api/java/time/package-summary.html).

I'd go with the latter.